### PR TITLE
Use AWS SDK 2

### PR DESCRIPTION
**Pull request by pieterjanpintens**
This PR updates the maven dependencies to AWS SDK 2.
It also moves from Apache log framework to slf4j.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/test/java/com/amazon/sqs/javamessaging/MessageListenerConcurrentOperationTest.java [hunk 1]
```diff
@@ -292,7 +288,7 @@ public void verify() {
 
     public void testConcurrentExecution(final MessageListener msgListener, final ConcurrentOperation operation) throws JMSException, InterruptedException {
 
-        long start = System.currentTimeMillis();
```
Comment (sfragata) [inline src/test/java/com/amazon/sqs/javamessaging/MessageListenerConcurrentOperationTest.java:295]: You can get rid of this line

**Comment by sfragata** [comment]
A little comment @pieterjanpintens , great job

**Comment by sfragata** [comment]
I'd change the pom.xml version from `1.0.8` to a 2.0 version with breaking changes (migrate aws lib from 1.X to 2.X)

**Comment by Thunderforge** [comment]
@robin-aws @haocheng-aws @troy-aws @kuba-aws Can any of you please review this excellent PR to upgrade to AWS SDK 2.x?

**Comment by robin-aws** [comment]
@adam-aws might also be able to help

**Comment by sfragata** [comment]
:cricket: :cricket: :cricket:

**Comment by danielpoplawskiacu** [comment]
Any info about when this could be merged and if it'll be ?

**Comment by gerhard-engelbrecht** [comment]
@robin-aws,  is anyone able to look at this please :)

**Comment by CarlosMOGoncalves** [comment]
Hello, 
I would just like to thank you for this PR.
Me and my team would very much like to see this improvement make it to release.
It would be tremendously helpful.
Cheers

**Comment by SunithaR** [comment]
Any word on when this PR will be reviewed and merged we are really looking for support for AWS SDK 2

**Comment by philmadden83** [comment]
@sfragata can we get this merged? Is there any support needed?

**Comment by SunithaR** [comment]
This has been sitting waiting to be merged for 2 years now :( @robin-aws @adam-aws

**Comment by CarlosMOGoncalves** [comment]
Hello again.
Anyone of the maintainers can have a look at this in order to merge it?
We would greatly appreciate to get rid of the SDK V1.
Thanks.

**Comment by GodwinMaravai** [comment]
Hello owner, 

When this will be merged and released? we are eagerly waiting for this lib?

**Comment by okohub** [comment]
Just here to up :)

+1

Please merge this.

**Comment by okohub** [comment]
@eddy-aws ping

**Comment by ziyanli-amazon** [comment]
I will review the commits.

**Pull request merged**